src/components/ArticlePage.jsx

```diff
@@ -60,20 +62,48 @@ export default function ArticlePage() {
       });
   }
 
+  function handleCommentClick(event) {
+    toggle ? isToggled(false) : isToggled(true);
+  }
+
+  function addNewCommentToList(comment) {
+    fetchComments(articleId).then(() => {
+      comment.votes = 0;
+      setComments((currentComments) => {
+        return [comment, ...currentComments];
+      });
+    });
+  }
```

Comment:
You shouldn't need to do fetchComments again here, if your useEffect is already fetching the comments and setting them as state then you have access to currentComments already through setComments, then a change in the state should cause a re-render